stubs/caldav/caldav/davclient.pyi

```diff
@@ -20,7 +20,7 @@ class DAVResponse:
     status: int
     headers: CaseInsensitiveDict[str]
     objects: dict[str, dict[str, str]]  # only defined after call to find_objects_and_props()
```

Comment:
```suggestion
    objects: dict[str, dict[str, str]]  # only defined after call to find_objects_and_props()
    huge_tree: bool
```

Reply:
This is marked as resolved, but you haven't applied the change

Reply:
Oh, it is because I was preparing https://github.com/python-caldav/caldav/pull/323 :)
I will change it now.